stack.yaml

```diff
@@ -59,7 +59,8 @@ extra-lib-dirs:
 # arch: x86_64
 #
 # Extra directories used by stack for building
-# extra-include-dirs: [/path/to/dir]
+extra-include-dirs:
+    - /Library/Developer/CommandLineTools/SDKs/MacOSX12.3.sdk/usr/include/ffi/
```

Comment:
We cannot have this in, it is a local directory on your machine.

Reply:
I don't have a better solution for that atm. Do you?